### ISSUE TITLE
How to use QKV2Context plugin in onnx

### ISSUE THREAD

**Issue by triple-Mu** [NVIDIA/TensorRT#2562]
## Description

![image](https://user-images.githubusercontent.com/92794867/208939249-c641b907-e482-49f1-a1bc-89a5da518cff.png)
How does QKV2Context work? How are the inputs arranged? Can you give the relevant pytorch execution code?
I want to use this plugin in bert onnx. What operators should I replace?
## Environment

**TensorRT Version**: 
**NVIDIA GPU**: 
**NVIDIA Driver Version**: 
**CUDA Version**: 
**CUDNN Version**: 
**Operating System**: 
**Python Version (if applicable)**: 
**Tensorflow Version (if applicable)**: 
**PyTorch Version (if applicable)**: 
**Baremetal or Container (if so, version)**:

**Comment by triple-Mu** [comment]
How to use bert onnx to register qkv2context plugin, whether there are original onnx and onnx files registered with plugin.

**Comment by zerollzeng** [comment]
If you are mean https://github.com/NVIDIA/TensorRT/tree/main/plugin/bertQKVToContextPlugin. It should be used by https://github.com/NVIDIA/TensorRT/tree/main/demo/BERT

**Comment by triple-Mu** [comment]
> If you are mean https://github.com/NVIDIA/TensorRT/tree/main/plugin/bertQKVToContextPlugin. It should be used by https://github.com/NVIDIA/TensorRT/tree/main/demo/BERT
What's the meaning of 'input_mask'?

**Comment by zerollzeng** [comment]
@ttyio ^ ^

**Comment by ttyio** [comment]
It's `mask` input preprocessed and outputted by plugin `embLayerNormPlugin` https://github.com/NVIDIA/TensorRT/tree/main/plugin/embLayerNormPlugin

**Comment by ttyio** [comment]
closing since no activity for more than 3 weeks, please reopen if you still have question, thanks!
